### ISSUE TITLE
Buffer progress bar is almost unnoticeable. Suggestion: orange buffer

### ISSUE THREAD

**Issue by allvoxman** [Chocobozzz/PeerTube#610]
Hey!

I've been playing and uploading to PeerTube a few weeks now, and I'd like to suggest this idea based on my experience. 

Ok, buffering progress bar is so faint that it's almost invisible. That can be annoying sometimes, for instance when buffering stops and you don't know where. Since orange seems to be a brand color here, I feel that it could be great if the buffering bar were orange instead of the current faint gray/transparent bar.

Thanks for this amazing project and keep up the good work!

**Comment by Chocobozzz** [comment]
Hello @allvoxman 

In fact the load progress was wrong: we don't have buffer logic in PeerTube and we try to download all the possible data, to quickly become a seeder. I fixed it in https://github.com/Chocobozzz/PeerTube/commit/77728efa627527a9f0f27010a597e8984d7a27f8. I tried to add more contrast between the loading bar and the placeholder. It's not orange because I'm not sure it would be very beautiful. But administrators can choose the color they want :)
